.github/workflows/image-push-release.yml

```diff
@@ -35,9 +35,11 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           context: .
+          platforms: linux/amd64
           push: true
           tags: |
-            ${{ steps.docker_meta.outputs.tags }}
+            ${{ steps.docker_meta.outputs.tags }}-amd64
+            ${{ env.IMAGE_NAME }}:${{ github.sha }}
```

Comment:
this is not needed.

we only create a tag with sha for push to master only

Reply:
Thanks!